contrib/tesseract/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "tesseract"
+pkgver = "5.4.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DUSE_SYSTEM_ICU=ON",
+    "-DBUILD_SHARED_LIBS=ON",
+    "-DTESSDATA_PREFIX=/usr/share",
+]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+makedepends = [
+    "icu-devel",
+    "leptonica-devel",
+    "libarchive-devel",
+    "libcurl-devel",
+    "pango-devel",
+]
+depends = [
+    "tesseract-data-eng",
+    "tesseract-data-osd",
+]
+pkgdesc = "OCR engine"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "Apache-2.0"
+url = "https://tesseract-ocr.github.io"
+source = f"https://github.com/tesseract-ocr/tesseract/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "c4bc2a81c12a472f445b7c2fb4705a08bd643ef467f51ec84f0e148bd368051b"
+
+
+@subpackage("libtesseract")
```

Comment:
is there some reason it has to be called libtesseract instead of tesseract-libs?

Reply:
sure

Reply:
though thinking about it this doesn't matter that much so i'd rather use tesseract-libs/tesseract-devel here anyway